android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -53,14 +55,28 @@ class DVCClient private constructor(
     private val configRequestQueue = ConcurrentLinkedQueue<UserAndCallback>()
     private val configRequestMutex = Mutex()
 
+    private var latestIdentifiedUser: User = user
+
     init {
         initializeJob = coroutineScope.async(coroutineContext) {
             isExecuting.set(true)
             try {
                 fetchConfig(user)
                 isInitialized.set(true)
                 withContext(Dispatchers.IO){
-                    eventSource = EventSource.Builder(Handler(fun () {}), URI(config?.sse?.url)).build()
+                    eventSource = EventSource.Builder(Handler(fun (event: String?, messageEvent: MessageEvent?) {
+                        val data = JSONObject(messageEvent?.data)
+                        var type = ""
+                        try {
+                            type = data.get("type") as String
+                        } catch (e: Exception) {}
+
+                        println("!! in message callback: type: $type query: ${messageEvent?.origin?.query}") // TODO remove
+
+                        if (type == "refetchConfig" && messageEvent?.origin?.query?.contains("sse=1") == true) { // TODO not sure if 'origin' is where the query param would show up?
```

Comment:
this should just check if the type is refetchConfig or the type is not set.

The SSE param is sent by the SDK to the config API during the refetchConfig operation (which should also be happening here)